db.go

```diff
@@ -737,14 +738,19 @@ func (db *DB) get(key []byte) (y.ValueStruct, error) {
 		if vs.Meta == 0 && vs.Value == nil {
 			continue
 		}
+		found = true
 		// Found the required version of the key, return immediately.
 		if vs.Version == version {
+			y.NumGetsWithResultsAdd(db.opt.MetricsEnabled, 1)
```

Comment:
Is this where it finds a value in the memtables without looking in the SST file structures? If so the metric seems it should be "NumGetsInMemtable" or similar. (I may not be reading this code right, however).